app/controllers/concerns/flash_i18n.rb

```diff
@@ -0,0 +1,30 @@
+module FlashI18n
+  protected
+
+  def redirect_to(url, options = {})
+    self.class._flash_types.each do |flash_type|
```

Comment:
I'm... not wild about it living in the app codebase. But in general I'm not wild about overriding library/framework methods -- I prefer wrapping them so it's immediately obvious to the developer that they're custom methods.

However, you're the one that'll be working on it with the rest of the epets team, so it's ultimately your call :)